types/jsrsasign/modules/KJUR/crypto/MessageDigest.d.ts

```diff
@@ -91,7 +91,7 @@ declare namespace jsrsasign.KJUR.crypto {
          * @example
          * md.digest()
          */
-        digest(): void;
+        digest(): string;
```

Comment:
I think digestString and digestHex's return types should also be updated to string:

https://kjur.github.io/jsrsasign/wikistatic/Tutorial-for-MessageDigest-class.html
https://kjur.github.io/jsrsasign/api/symbols/KJUR.crypto.MessageDigest.html

Reply:
Added